src/components/notifications/SMSPartner.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div class="mb-3">
+        <label for="smspartner-key" class="form-label">{{ $t("API Key") }}</label>
+        <HiddenInput id="smspartner-key" v-model="$parent.notification.smspartnerApikey" :required="true" autocomplete="new-password"></HiddenInput>
+        <div class="form-text">
+            <i18n-t keypath="smspartnerApiurl" as="div" class="form-text">
+                <a href="https://my.smspartner.fr/dashboard/api" target="_blank">https://my.smspartner.fr/dashboard/api</a>
+            </i18n-t>
+        </div>
+    </div>
+    <div class="mb-3">
+        <label for="smspartner-phone-number" class="form-label">{{ $t("smspartnerPhoneNumber") }}</label>
+        <input id="smspartner-phone-number" v-model="$parent.notification.smspartnerPhoneNumber" type="text" minlength="3" maxlength="20" pattern="^[\d+,]+$" class="form-control" required>
+        <div class="form-text">
+            <i18n-t keypath="multiple numbers must be separated by" as="div" class="form-text">
```

Comment:
Fuckups happen, should forgot to add this comment ^^

```suggestion
            <i18n-t keypath="smspartnerPhoneNumberHelptext" as="div" class="form-text">
```